dnet/dnet_main.h

```diff
@@ -6,6 +6,8 @@
 struct send_parameters {
         void *connection;
         time_t time_limit;
+        int broadcast;
+        uint8_t ttl;
```

Comment:
What is broadcast and ttl? 
connection and time_limit have describing name, but new variables - not. Especially 'ttl'